eng/testing/tests.targets

```diff
@@ -28,10 +28,16 @@
   <Target Name="ArchiveTests"
           Condition="'$(ArchiveTests)' == 'true' and '$(IgnoreForCI)' != 'true' and
                      ('$(TargetsMobile)' != 'true' or '$(TargetOS)' == 'Browser')"
-          AfterTargets="Build"
+          AfterTargets="PrepareForRun"
```

Comment:
The reason why I changed this was because this will run twice and the second time it runs it will overwrite the .zip with the wrong .zip.

The reason for that is because we have an inner and outerbuild.

Reply:
Actually it was because: the ArchiveTests target has an `AfterTargets=PrepareForRun` which is executed inside `Build` target... our publish target has an `AfterTargets=Build` because `Publish` depends on Build; so for browser, we were essentially zipping up the folder before publishing the app and building the bundle, resulting in an empty zip.

Reply:
Ah I see, that makes sense :)

Reply:
I fixed it to not have the circular dependency now 😄